modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -181,6 +182,58 @@ public ObjectValidationRule updateObjectValidationRule(
 		return objectValidationRulePersistence.update(objectValidationRule);
 	}
 
+	@Override
+	public void validate(
+			Object object, long objectDefinitionId,
+			Map<String, Object> modelAttributes)
+		throws PortalException {
```

Comment:
Hi @seltong,

try to merge this method with the other one for ObjectEntry by using  `instanceof`.

Thanks!